component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -847,6 +847,57 @@ public Response sendOnBoardingEmail(@Context HttpServletRequest request,
     sendOnBoardingEmail((UserImpl) user, url);
     return Response.ok().build();
   }
+
+  @POST
+  @Path("multiSelection/{action}")
+  @Produces(MediaType.APPLICATION_JSON)
+  @RolesAllowed("administrators")
+  @ApiOperation(value = "Onboard, enable ou disable a specific users", httpMethod = "POST", response = Response.class, notes = "This will enroll, enable ou disable a specific users.")
```

Comment:
```suggestion
  @ApiOperation(value = "Make action on list of users", httpMethod = "PATCH", response = Response.class, notes = "This will realize the action on the list of users if possible")
```